templates/lxc-fedora.in

```diff
@@ -812,6 +812,13 @@ setup_repositories()
     local release="${3}"
     local mirror="${4}"
 
+    PKGS=(fedora-release-${release} fedora-repos-${release}
```

Comment:
the opening bracket '(' breaks the whole script ;)

Reply:
We need a signed-off by before we can even look at the code. :(

Reply:
well, at least for me it doesn't matter, as the script (even with the line fixed with quotes) still isn't able to get me a F28 container on a debian host ;)